PROTO_tests/tests/numeric_test.py

```diff
@@ -3,7 +3,8 @@
 import arkouda as ak
 from arkouda.dtypes import npstr
 
-prob_size = 1000
+#prob_size = 1000
+prob_size = 10
```

Comment:
it looks like this was commented out for testing, but I think we should prob put it back to `1000`